lib/slimlock.nix

```diff
@@ -101,8 +101,8 @@
         mkdir -p $out/js
         cd $out/js
         cp -r $src/. .
-        cat ${tarballsFile} | xargs npm cache add
-        npm ci ${omitCmd omit} ${auditCmd audit} --ignore-scripts
+        cat ${tarballsFile} | xargs -L 1 npm cache add
```

Comment:
Previously, not all tarbals were added to cache. I checked that with `npm cache ls`.

Then, in `npm help cache`, I saw `npm cache add <package-spec>`. So, I decided that it expects a single tarball and added `-L 1`.